src/makeRobot.js

```diff
@@ -39,6 +39,67 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    n: name,
+    v: version,
```

Comment:
don't use one-letter-variables:
```suggestion
    name,
    version,
```